styleguide/_themes/derek/scss/components/_ceiling.scss

```diff
@@ -44,13 +46,11 @@ $ceiling-font-size: 11px;
     margin-bottom: 0;
 
     > li{
-      margin: $ceiling-margin 0;
-      padding: 0;
-      &:after {
-        content: "|";
-      }
-      &:last-child::after {
-        content: " ";
+      margin: floor($ceiling-margin + 2) 0;
```

Comment:
Is this floor necessary? Isn't ceiling-margin already an integer, based on its definition already using floor(): 
```$ceiling-margin: floor($navbar-margin-bottom / 4);```

I suppose it doesn't hurt anything as it all gets compiled down anyway, but just thought I'd point it out.